quantum/quantum.c

```diff
@@ -51,14 +51,6 @@ float default_layer_songs[][16][2] = DEFAULT_LAYER_SONGS;
 #    endif
 #endif
 
-#ifdef AUTO_SHIFT_ENABLE
-#    include "process_auto_shift.h"
```

Comment:
was the removal of this intentional? 
(eg, autoshift's header include)

Reply:
Its not required as its included elsewhere in `quantum.h`. I guess it could be seen as out of scope of this PR.

Reply:
Ah, just wanted to make sure.